src/Agent.Worker/CodeCoverage/FeatureFlagUtility.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.VisualStudio.Services.Agent.Util;
+using Microsoft.VisualStudio.Services.FeatureAvailability.WebApi;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.CodeCoverage
+{
+    public static class FeatureFlagUtility
+    {
+        public static bool GetFeatureFlagState(FeatureAvailabilityHttpClient featureAvailabilityHttpClient, string FFName, IAsyncCommandContext context)
+        {
+            try
+            {
+                var featureFlag = featureAvailabilityHttpClient?.GetFeatureFlagByNameAsync(FFName).Result;
+                if (featureFlag != null && featureFlag.EffectiveState.Equals("On", StringComparison.OrdinalIgnoreCase))
```

Comment:
the condition should be that if it is off then return false. in any other condition return true